docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -485,6 +486,7 @@ Valid options for `[Kube]` are listed below:
 | ConfigMap=/tmp/config.map        | --config-map /tmp/config.map           |
 | LogDriver=journald               | --log-driver journald                  |
 | Network=host                     | --net host                             |
+| PodmanArgs=--annotation=key=value | --annotation=key=value                 |
```

Comment:
Can you reformat the table to align the separators?

Reply:
I'm trying to be careful with pushing it further and further. But, I guess because it's just one space, it's ok. Done